packages/backend/src/utils/parsers/quadlet-unit-parser.ts

```diff
@@ -56,6 +72,42 @@ export class QuadletUnitParser extends Parser<string, Quadlet> {
     return `${source['Install']['DefaultInstance']}`;
   }
 
+  protected toAbsolute(sourcePath: string, path: string): string {
+    if (isAbsolute(path)) return path;
+    if (path.startsWith('~')) return path;
+
+    return join(dirname(sourcePath), path);
+  }
+
+  protected findResources(sourcePath: string, source: IIniObject): Array<FileReference> {
+    return Object.entries(QUADLET_RESOURCES_PATHS).reduce((accumulator, [quadletType, properties]) => {
```

Comment:
suggestion: at this point we already know the quadlet type, so we could pass it to this function to avoid looping on all entries.